frontend/src/useFlow.ts

```diff
@@ -205,35 +228,72 @@ const createSources = async (name: string, originalLanguage: string, translatedL
     original_source_id: originalSource.id,
```

Comment:
This field should be deleted and we should only connect sources via the new table.